parity/rpc_apis.rs

```diff
@@ -566,6 +570,13 @@ impl ApiSet {
 				public_list.insert(Api::Personal);
 				public_list
 			},
+			ApiSet::PubSub => vec![
```

Comment:
definitely not a bottleneck in startup -- but cloning N 1-byte enum values should be really quick.